mantidproject/mantid
# Make unsaved changes pop-up appear once in ISIS Reflectometry Polref Interface

In ISIS Reflectometry Polref the user should be asked once whether they want to save or discard their unsaved changes when loading in a new table to the Reflectometry (Polref) interface. However this pop-up was appearing twice if the user chose not to discard their changes.

**To test:**
- Go to `Interfaces > Reflectometry > ISIS Reflectometry (Polref)`
- Import `INTER_NR.TBL` from SampleData-ISIS (`Reflectometry > Import .TBL`)
- Remove all of the rows in the processing table and insert just one row
- Fill in some of the columns in the processing table.
- Go to `Reflectometry > Open Table > Your TBL File`
- See that the unsaved changes warning only appears once.
- Ensure that all other functionality of the TBL regarding loading/saving table still works properly.

Fixes #16250
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
